### ISSUE TITLE
paella-core 1.3 has a broken progress bar

### ISSUE THREAD

**Issue by LukasKalbertodt** [polimediaupv/paella-core#108]
I just updated Paella in Tobira to 1.3 and the bug #97 is in fact fixed. However, the progress bar seems broken. Try any video from here: https://update-paella.tobira.opencast.org/ 

The progress bar is not shown and in the console, there are tons of errors:

```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'clientWidth')
```

I checked the recent commits and found this one which adds two lines reading `clientWidth`: https://github.com/polimediaupv/paella-core/commit/dae79c9494452c1b93b402b41674c69d2719cd2f#diff-2b5cb655d67d7cb5f86eda099af8ca18eb615c4435ee16ddca930760f70d6c00R136-R137

**Comment by ferserc1** [comment]
Fixed in version 1.3.1

**Comment by LukasKalbertodt** [comment]
I can confirm this fixes the bug. Thanks for the quick release!
